SemiTruck.h

```diff
@@ -1,2 +1,14 @@
 #pragma once
 
+#include "Vehicle.h"
+
+struct SemiTruck : public Vehicle
+{
+    SemiTruck(const std::string& n);
+
+    ~SemiTruck() override;
+    SemiTruck(const SemiTruck&);
+    SemiTruck& operator = (const SemiTruck&);
+
+    void pullOver();
```

Comment:
WHere's the function to call when they enter the highway? 
- [ ] revise

Reply:
Added SemiTruck::cruiseInRightLane()